src/main/java/lotto/domain/WinningResult.java

```diff
@@ -0,0 +1,30 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class WinningResult {
```

Comment:
`List<WinningType>`를 가지는 것으로 보이네요, 크게 중요한 내용은 아니지만 클래스에도 복수의 결과가 포함될 수 있음을 나타내주는건 어떨까요?

```suggestion
public class WinningResults {
```

Reply:
넵 그렇네요 
꼼꼼히 봐주셔서 감사합니다 😁👍